docs/docs-content/release-notes/release-notes.md

```diff
@@ -75,10 +75,11 @@ to learn more about the changes introduced in this release.
 
 #### Deprecations and Removals
 
-- The [cluster group](../clusters/cluster-groups/cluster-groups.md), Beehive, will be sunset on November 9, 2024. If you
-  are using Palette SaaS and have virtual clusters in the Beehive cluster group, migrate the workload to new virtual
-  clusters hosted in a self-managed cluster group before November 9, 2024. You can learn more about creating a new
-  cluster group in the [Create and Manage Cluster Groups](../clusters/cluster-groups/create-cluster-group.md) guide.
+- The [cluster group](../clusters/cluster-groups/cluster-groups.md), Beehive, will be sunset on November 9, 2024. As of
+  this release, you are no longer able to deploy any new virtual clusters into Beehive. If you are using Palette SaaS
```

Comment:
I would put release number here, so the user doesn't have to scroll up to see which version we are referring to.